dashboard/dashboard-impl/pom.xml

```diff
@@ -2,16 +2,19 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
     <parent>
-        <groupId>org.text-adventure.quest-editor</groupId>
-        <artifactId>quest-editor</artifactId>
```

Comment:
And why are change quest-editor module?
I think you need revert all other's files. 
Commit your and only your files and module